mypy.ini

```diff
@@ -3,6 +3,27 @@ files = ignite
 pretty = True
 show_error_codes = True
 
+check_untyped_defs = True
+; a lot of work needed to fix issues
```

Comment:
Could you please detail this point ? Is it only for `disallow_any_generics = False` to make it `True`, right ?

Reply:
Yeah, if I set `disallow_any_generics = True` a lot of "errors" will pop up, like all the `Callable` types have to be more precise.